src/ClientOptions.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Utopia\Mongo;
 
-class MongoClientOptions
+class ClientOptions
```

Comment:
This class is over complicating things. We can just use a regular signature in the constructor instead of forcing another class. This is also more common to PHP

Reply:
Done